microsoft/testsuites/adhoc/tool_install.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, List, Union
+
+from lisa import (
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    tools,
+)
+from lisa.executable import Tool
+from lisa.util import LisaException, SkippedException
+
+
+@TestSuiteMetadata(
+    area="adhoc",
+    category="functional",
+    description="""
+    This test suite includes adhoc tests
+    """,
+)
+class Adhoc(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will install the tools
+        passed as parameter 'install_tools'
+        """,
+        priority=5,
+    )
+    def tool_install(
+        self, log: Logger, node: RemoteNode, variables: Dict[str, Any]
+    ) -> None:
+        tool_name_parameter = "install_tools"
```

Comment:
how about `case_tool_install`? So it's easy to identify where the variable is used for.